main.js

```diff
@@ -136,41 +149,54 @@ function changeTimerColor() {
 
 function startTimer() {
   startTimerButton.disabled = true;
-  setInterval(function () {
+  var interval = setInterval(function () {
     if (currentActivity.completed === false) {
-      currentActivity.countdown()   
-      //   if (currentActivity.completed === true) {
-    //     showElement(logActivityButton)
-    //   }
-    // }
+      currentActivity.countdown()
+    } else {
+      clearInterval(interval);
     }
   }, 1000)
 };
 
 
 
-function showMyLogs() {
-  activityCards.forEach(card => {
-    pastActivityCards.innerHTML = `
-    <div class="card-container">
-      <div class="card-header">
-        <p class="card-category">${currentActivity.category}</p>
-        <p class="card-duration">${currentActivity.minutes} MIN   :   ${currentActivity.seconds} SEC</p>
-        <p class="card-body">${currentActivity.description}</p>
+function saveActivityToLocalStorage() {
+  localStorage.setItem(`${currentActivity.id}`, JSON.stringify(currentActivity));
+  pastActivityCards.innerHTML = '';
+  showMyLog();
+}
+
+
+function showMyLog() {
+  for (i = 0; i < localStorage.length; i++) {
+    pastActivityCards.innerHTML += `
+    <div class='card-container'>
+      <div class='card-header'>
+        <p class='card-category'>${JSON.parse(localStorage.getItem(localStorage.key(i))).category}</p>
+        <p class="card-duration">${JSON.parse(localStorage.getItem(localStorage.key(i))).savedMinutes} MIN : ${adjustSeconds(JSON.parse(localStorage.getItem(localStorage.key(i))).savedSeconds)} SEC</p>
+        <p class="card-body">${JSON.parse(localStorage.getItem(localStorage.key(i))).description}</p>
+      </div>
+        <div class = "sliver-${JSON.parse(localStorage.getItem(localStorage.key(i))).category}">
```

Comment:
What if the user has anything else in their local storage? won't it pull that out, too?